rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -832,6 +832,7 @@ stream {
         {{ template "CUSTOM_ERRORS" (buildCustomErrorDeps $errorLocation.UpstreamName $errorLocation.Codes $all.EnableMetrics) }}
         {{ end }}
 
+        {{ buildMirrorLocations $server.Locations }}
```

Comment:
Where're the hardcoded mirror directives here? I was expecting to see them being deleted from this file.

Reply:
The mirror configuration is inside the location block